docs/conceptual/rvs-modules.rst

```diff
@@ -0,0 +1,88 @@
+.. meta::
+  :description: Video decoding pipeline
+  :keywords: video decoder, video decoding, rocDecode, AMD, ROCm
+
+***********************************
+ROCmValidationSuite Architecture
```

Comment:
Isn't supposed to be ROCm Validation Suite  (with spaces)

Reply:
Shouldn't these match?
![image](https://github.com/ROCm/ROCmValidationSuite/assets/164940351/30e4fd81-8415-424a-9a58-f590862ee8fc)

Reply:
Fixed. Thanks, Leo.

Reply:
Fixed. 
<img width="710" alt="rvsmodules" src="https://github.com/ROCm/ROCmValidationSuite/assets/56051583/199602f0-5ed0-4c46-b9cd-057bb3119430">

Reply:
> Isn't supposed to be ROCm Validation Suite (with spaces)

Fixed.

Reply:
> Isn't supposed to be ROCm Validation Suite (with spaces)

Fixed